### PR TITLE
feat(capacity): remove target validity

### LINKED ISSUE #894: [Capacity] Epoch Period can be set through governance
### Description:
As an stakeholder of the frequency chain I should be able to submit a proposal to update the epoch period length.    

Acceptance Criteria:

- I should be able to submit an extrinsic that updates the period.
- Origin is Root.
- Sets the new Epoch-Period.
- New Epoch-Period begins at the Next Epoch's start.

### PR DESCRIPTION
**Pull request by enddynayn**
# Goal
Remove target validity when unstaking.
This validation is not needed since a
target-staker relationship existence is
checked when attempting to reduce the Capacity
of a target.

Closes #894

### CODE REVIEW

**Comment by enddynayn** [comment]
Waiting for Jenkins to generate new weights.

File: pallets/capacity/src/weights.rs [hunk 1]
```diff
@@ -107,33 +105,31 @@ impl WeightInfo for () {
...
-	// Storage: Msa ProviderToRegistryEntry (r:1 w:0)
 	// Storage: Capacity StakingAccountLedger (r:1 w:1)
 	// Storage: Balances Locks (r:1 w:1)
 	// Storage: Capacity StakingTargetLedger (r:1 w:1)
 	// Storage: Capacity CapacityLedger (r:1 w:1)
 	fn unstake() -> Weight {
-		Weight::from_ref_time(52_608_000 as u64)
-			.saturating_add(RocksDbWeight::get().reads(5 as u64))
+		Weight::from_ref_time(43_549_000 as u64)
+			.saturating_add(RocksDbWeight::get().reads(4 as u64))
```
Comment (enddynayn) [inline pallets/capacity/src/weights.rs:132]: Reads are decreased by one as a result of removing target validation.

File: pallets/capacity/src/weights.rs [hunk 2]
```diff
@@ -107,33 +105,31 @@ impl WeightInfo for () {
...
 	}
 	// Storage: Capacity CurrentEpochInfo (r:1 w:1)
-	// Storage: Capacity CurrentEpoch (r:1 w:1)
 	// Storage: Capacity CurrentEpochUsedCapacity (r:0 w:1)
 	fn on_initialize() -> Weight {
-		Weight::from_ref_time(7_669_000 as u64)
-			.saturating_add(RocksDbWeight::get().reads(2 as u64))
-			.saturating_add(RocksDbWeight::get().writes(3 as u64))
+		Weight::from_ref_time(6_442_000 as u64)
+			.saturating_add(RocksDbWeight::get().reads(1 as u64))
```
Comment (enddynayn) [inline pallets/capacity/src/weights.rs:123]: Nice. We improved the reads and writes of on_intitialize in another pr.

**Review by shannonwells — APPROVED** [review]
do it to it

**Review by johnmela — CHANGES_REQUESTED** [review]
Looks good!

Let's also remove line 95 from `capacity/src/benchmarking.rs`
`register_provider::<T>(target, "Foo");`

**Review by johnmela — APPROVED** [review]

**Pull request merged**